test/specs/add.spec.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-len */
+/* jshint esversion: 6 */
+
+const Calculator = require('../../app/calculator');
+const {expect} = require('chai');
+
+describe(`positive tests for addition`, () => {
+  let calc;
+  beforeEach(() => {
+    calc = new Calculator();
+  });
+  afterEach(() => {
+    calc = null;
+  });
+
+  it(`shoud return summ of positive integer numbers`, () => {
```

Comment:
Please correct a misprint: **sum**. The same is for other tests.

Reply:
fixed